spec/datadog/tracing/contrib/kafka/patcher_spec.rb

```diff
@@ -56,6 +56,7 @@
           expect(span).to_not have_error
           expect(span.get_tag(Datadog::Tracing::Metadata::Ext::TAG_COMPONENT)).to eq('kafka')
           expect(span.get_tag(Datadog::Tracing::Metadata::Ext::TAG_OPERATION)).to eq('connection.request')
+          expect(span.get_tag(Datadog::Tracing::Contrib::Ext::Messaging::TAG_SYSTEM)).to eq('kafka')
```

Comment:
Prefer plain text to indirection in test

```suggestion
          expect(span.get_tag('messaging.system)).to eq('kafka')
```